util/include/Metrics.hpp

```diff
@@ -23,13 +24,20 @@
 #include "Statistics.hpp"
 
 namespace concordMetrics {
+template <class T>
+class basic_gauge;
```

Comment:
Nit: According to the code convention, classes have to be `PascalCase` => `BasicGauge`.

Reply:
done